### PR TITLE
installation: Add jupyter video, clarify installation some.

### PR DESCRIPTION
**Pull request by rkdarst**
- If someone can start JupyterLab, that is all they need (since it
  includes the terminal).  Rearrange instructions to emphasize
  jupyterlab-first.
- Review: general check + details

### CODE REVIEW

File: installation.rst [hunk 1]
```diff
@@ -26,45 +26,56 @@ we don't support them.
 Verify the installation
 ~~~~~~~~~~~~~~~~~~~~~~~
 
+.. admonition:: Watch the video
+
+   See this `verification in video form
+   <https://youtu.be/OEX1ss_HCHc>`__ - if you can do this, you are
```
Comment (annefou) [inline installation.rst:32]: Should we also mention Anaconda Navigator as another way to start jupyterLab. On my laptop, I very often use it to start jupyterLab: being on windows it is often the best way to get all the paths right.

File: installation.rst [hunk 2]
```diff
@@ -26,45 +26,56 @@ we don't support them.
...
+   ready to go for day one.
+
 After activating the conda environment, you should be able to check
 the Python version from the command line.  Any version of python3
```
Comment (annefou) [inline installation.rst:36]: Same as above. Using anaconda navigator can be another simple approach (no need to activate/deactivate).

File: installation.rst (hunk 1 above)
Comment (rkdarst) [reply installation.rst:32]: I'll add it to the text.  I don't use the navigitor and don't know if I can make a useful video showing starting jupyter via it..

Comment (annefou) [reply installation.rst:32]: I would not bother to make another video. I would simply add some kind of callout; we can also point to online documentation (https://docs.anaconda.com/anaconda/navigator/).

File: installation.rst (hunk 2 above)
Comment (rkdarst) [reply installation.rst:36]: I wasn't sure how to mention it here, given all the different types of shells that might be - what would you say?  Can you add the suggestion?  (I'd like to get this out as soon as possible)

What I settled on was recommending to start the terminal via jupyterlab not independently.

**Review by annefou — APPROVED** [review]
Excellent!

**Pull request merged**